core/templates/search/search_pages_results.html

```diff
@@ -112,18 +234,9 @@ <h4 class="term">{{ paginator.count }} results
       <input type="hidden" name="andtext" value="{{query.andtext}}" />
       <input type="hidden" name="phrasetext" value="{{query.phrasetext}}" />
 
-      {# location #}
-      <input type="hidden" name="city" value="{{query.city}}" />
-      <input type="hidden" name="county" value="{{query.county}}" />
-
       {# date range and years #}
-      {# if there is a yearRange, it supercedes the date ranges #}
-      {% if request.GET.yearRange %}
-        <input type="hidden" name="yearRange" value="{{query.yearRange}}" />
-      {% else %}
-        <input type="hidden" name="date1" value="{{query.date1}}" />
-        <input type="hidden" name="date2" value="{{query.date2}}" />
-      {% endif %}
+      <input type="hidden" name="date1" value="{{query.date1}}" />
```

Comment:
The logic for yearRange precedence needs to be copied here as well. If I filter with a year range, then do nothing more than click the "Refine Search" button on the resulting page, it clears that filter out because the hidden input isn't present for this `<form>`.

Reply:
Hmm, this is going to take a little more thought.  When you first go to the page, the hidden field sends an empty string when you filter, so then there are two yearRanges in the URL.  It appears to be continuing to pick up the empty one in successive requests, clearing out the yearRange filter.

I noticed this behavior with language, too, in my other branch, and just took out the hidden field.  That worked okay for language because it recognizes itself in the dropdown and sets the value again in the form on submission, but yearRange doesn't do this since the range changes.  I wonder if instead of building a form with "All" at the top or something, if we should have an option that IS the currently selected year range.  I don't know how hard that would be to build, or if it would make sense from the user's perspective, but I'll look into it a little bit to figure out how feasible it would be.

Reply:
Okay, the new commit I just pushed adds the yearRange selection to the current dropdown, which I think is probably a good idea regardless just so that users can get a visual confirmation of their selection besides in the description at the top.  I did figure out that the hidden field pattern of `query.proxdistance` etc doesn't seem to select correctly for `yearRange` but you CAN use `request.GET.yearRange` and that seems to work.  I don't think that the hidden field is necessary now that the current selection is in the dropdown, but it's available to us if we need it, I s'pose.